src/api.py

```diff
@@ -298,6 +299,61 @@ def forecast(lat, long, decimal, days=0):
     return forecast_data
 
 
+def get_hourly_forecast(lat, long, days=1):
+    """
+    Gets hourly weather data
+    """
+    # Setup the Open-Meteo API client with cache and retry on error
+    cache_session = requests_cache.CachedSession(".cache", expire_after=3600)
+    retry_session = retry(cache_session, retries=5, backoff_factor=0.2)
+    openmeteo = openmeteo_requests.Client(session=retry_session)
+
+    # Make sure all required weather variables are listed here
+    # The order of variables in hourly or daily is important
+    # to assign them correctly below
+    url = "https://api.open-meteo.com/v1/forecast"
+    params = {
+        "latitude": lat,
+        "longitude": long,
+        "hourly": ["cloud_cover", "visibility"],
+        "temperature_unit": "fahrenheit",
```

Comment:
It would be nice to pass in an argument that specifies the temperature unit. We can default to Fahrenheit. If the user passes the `metric` argument, Celsius should be shown.

See the `ocean_information` function to get an idea